pkgs/top-level/all-packages.nix

```diff
@@ -19017,6 +19017,12 @@ with pkgs;
     boost = boost160;
   };
 
+  kicad-unstable = callPackage ../applications/science/electronics/kicad/unstable.nix {
```

Comment:
`python.pkgs.callPackage`

Reply:
KiCad optionally uses python for scripting. It is not a python application.
Is this change still required?
Also no other package in `top-level.nix` uses the proposed code.

Reply:
there are a few that use `pythonPackages.callPackage` which is the same.

Yes, this should be used. Top-level packages should not have any `inherit (pythonPackages) foo;` as that risks mixing Python 2 and 3 packages (which was a common issue 3+years ago).

Reply:
Then I 'll use `pythonPackages.callPackage`for uniformity. Unless you object.
If you object, I propose to also change the others in a separate PR.